beacon_node/beacon_chain/src/schema_change/migration_schema_v20.rs

```diff
@@ -0,0 +1,108 @@
+use crate::BeaconChainTypes;
+use slog::{info, Logger};
+use std::sync::Arc;
+use store::{get_key_for_col, DBColumn, Error, HotColdDB, KeyValueStore, KeyValueStoreOp};
+use types::{Hash256, Slot};
+
+/// Chunk size for freezer block roots in the old database schema.
+const OLD_SCHEMA_CHUNK_SIZE: u64 = 128;
+
+fn old_schema_chunk_key(cindex: u64) -> [u8; 8] {
+    (cindex + 1).to_be_bytes()
+}
+
+pub fn upgrade_to_v20<T: BeaconChainTypes>(
+    db: Arc<HotColdDB<T::EthSpec, T::HotStore, T::ColdStore>>,
+    log: Logger,
+) -> Result<Vec<KeyValueStoreOp>, Error> {
+    info!(log, "Upgrading freezer database schema");
+    upgrade_freezer_database::<T>(&db, &log)?;
+
+    // No hot DB changes
+    return Ok(vec![]);
```

Comment:
@michaelsproul thoughts on migrating all finalized blocks? If we just delete that's going to be a lot of bandwidth on the entire network to backfill